test.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -e
+echo "" > coverage.txt
+
+echo "==> Running unit tests"
+for d in $(go list ./... | grep -v vendor); do
+    go test -race -coverprofile=profile.out -covermode=atomic "$d"
+    if [ -f profile.out ]; then
+        cat profile.out >> coverage.txt
+        rm profile.out
+    fi
+done
+
+echo "==> Running integration tests (./tests)"
+find . -path ./vendor -prune -o -name "*.sock" -exec rm {} \;
+make install
+bash tests/test.sh
```

Comment:
actually, we can call `make test_integrations` here

Reply:
dont we want `test_integrations` to run the shell scripts too tho ?

Reply:
oh, i see. ok